lib/tasks/cms/cms.rake

```diff
@@ -210,6 +210,11 @@ namespace :cms do
         puts "\t++ Fixing pagination href in #{page.id}/shared/pagination"
       end
     end
+
+    desc 'Fixes empty titles in sections'
+    task :section_empty_titles => :environment do
+      CMS::Section.unscoped.where(title: ['', nil]).update_all("title = system_name")
```

Comment:
Just FYI this probably needs to be added in release notes or upgrade notes so that customers can fix their databases.